client/mm/mm_basic.go

```diff
@@ -83,7 +98,32 @@ type MarketMakingConfig struct {
 
 	// EmptyMarketRate can be set if there is no market data available, and is
 	// ignored if there is market data available.
-	EmptyMarketRate float64 `json:"manualRate"`
+	EmptyMarketRate float64 `json:"emptyMarketRate"`
+
+	// SplitTxAllowed indicates whether the wallet (if it supports multi-splits)
+	// is allowed to do multi split transactions when funding orders. For UTXO
+	// based assets, it may not be possible to fund each of the orders without
+	// a split transaction which creates outputs of the size required by each
+	// order.
+	SplitTxAllowed bool `json:"splitTxAllowed"`
+
+	// SplitBuffer indicates whether the quote asset wallet (if it supports
+	// multi-splits) should add a buffer to the split amount. This is useful
+	// to prevent too many split transactions from being created. During the
+	// operation of this market maker, orders will be frequently placed and
+	// canceled as the price moves. If the price moves upward and there is
+	// no split buffer, the wallet may need to create a new split transaction
+	// to fund the new orders.
+	//
+	// The split buffer is a percentage of the total amount required for an
+	// order. For example if the order requires 0.1 BTC and the split buffer
+	// is 5, the wallet will create a split transaction with an output of
+	// 0.105 BTC.
+	SplitBuffer *uint64 `json:"splitBuffer"`
```

Comment:
This is a little off for me. The option to use splits should be a per-asset option. And EVM assets won't use this at all. These really are better represented using the `settings map[string]string` system that we had in place for asset-specific options.

What do you think of https://github.com/decred/dcrdex/compare/master...buck54321:multifundopts ?

Reply:
It is better like this, but it would be nice to block people from setting a buffer on the quote asset.

Reply:
Add a `QuoteOnly bool` field to `ConfigOption`?

Reply:
I thought of that too.. kind of out of place over there since it's just specific to a market making strategy. Since assets share a lot of these configs, maybe the keys could be defined in `client/asset/interface.go`, and then based on the key, the market maker could make sure it's always 0 for the base asset.

Reply:
> the keys could be defined in client/asset/interface.go, and then based on the key, the market maker could make sure it's always 0 for the base asset.

That a step backwards, imo. Let the assets decide. I'm all for adding fields to `ConfigOption`. It's a free-for-all there. Use `omitempty` so we're not bloating the results.